pkg/liqoctl/network/handler.go

```diff
@@ -63,6 +64,39 @@ func NewOptions(localFactory *factory.Factory) *Options {
 	}
 }
 
+func (o *Options) CheckAlreadyEstablished(ctx context.Context) (bool, error) {
+	ctx, cancel := context.WithTimeout(ctx, o.Timeout)
+	defer cancel()
+
+	// Create and initialize cluster 1.
+	cluster1, err := NewCluster(ctx, o.LocalFactory, o.RemoteFactory, true)
+	if err != nil {
+		return false, err
+	}
+
+	// Create and initialize cluster 2.
+	cluster2, err := NewCluster(ctx, o.RemoteFactory, o.LocalFactory, true)
+	if err != nil {
+		return false, err
+	}
+
+	_, err = cluster1.GetGatewayServer(ctx, forge.DefaultGatewayClientName(cluster2.localClusterID))
+	if err == nil {
+		return true, nil
+	} else if client.IgnoreNotFound(err) != nil {
+		return false, err
+	}
+
+	_, err = cluster2.GetGatewayClient(ctx, forge.DefaultGatewayServerName(cluster1.localClusterID))
+	if err == nil {
+		return true, nil
+	} else if client.IgnoreNotFound(err) != nil {
+		return false, err
+	}
```

Comment:
I would check for both client and server on both clusters

Reply:
It would become impossible to change network parameters using liqoctl peer without unpeer